### PR TITLE
Rizzcomic: Updated domain

### LINKED ISSUE #2590: Rizzcomic URL changed
### Source information

Rizz Comic 1.4.32

### Source language

English 

### Source new URL

https://rizzfables.com/

### Other details

Old URL of Rizzcomic is now changed to Rizzfables.com. which has made current extension to stop functioning and giving error 404. Opening in webview redirects to new website.

### Acknowledgements

- [X] I have updated all installed extensions.
- [x] I have opened WebView and checked that the source URL has not changed yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Mondotosz**
closes https://github.com/keiyoushi/extensions-source/issues/2590

I've never done mobile development nor worked with Kotlin so it took a while to test it but I built the apk and installed it on my phone to check if it actually works and it does.

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/en/rizzcomic/build.gradle [hunk 1]
```diff
@@ -2,8 +2,8 @@ ext {
     extName = 'Rizz Comic'
     extClass = '.RizzComic'
     themePkg = 'mangathemesia'
-    baseUrl = 'https://rizzcomic.com'
-    overrideVersionCode = 2
+    baseUrl = 'https://rizzfables.com/'
```
Comment (bapeey) [inline src/en/rizzcomic/build.gradle]:
```suggestion
    baseUrl = 'https://rizzfables.com'
```

File: src/en/rizzcomic/src/eu/kanade/tachiyomi/extension/en/rizzcomic/RizzComic.kt [hunk 2]
```diff
@@ -23,7 +23,7 @@ import java.util.Locale
 
 class RizzComic : MangaThemesiaAlt(
     "Rizz Comic",
-    "https://rizzcomic.com",
+    "https://rizzfables.com/",
```
Comment (bapeey) [inline src/en/rizzcomic/src/eu/kanade/tachiyomi/extension/en/rizzcomic/RizzComic.kt]:
```suggestion
    "https://rizzfables.com",
```

**Review by bapeey — COMMENTED** [review]
Remove trailing slash

File: src/en/rizzcomic/build.gradle (hunk 1 above)
Comment (Mondotosz) [reply src/en/rizzcomic/build.gradle]: I just noticed and pushed a commit addressing that 😅

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**